core/scheduler/scheduler.go

```diff
@@ -115,7 +115,7 @@ func (s *Scheduler) Run() error {
 
 			instrumentSlot(slot)
 
-			go s.emitCoreSlot(ctx, slot)
+			s.emitCoreSlot(ctx, slot)
```

Comment:
Why is this not executed in a goroutine anymore?

Reply:
yeah because `emitCoreSlot` method already calls each subscriber in a separate go routine: https://github.com/ObolNetwork/charon/blob/b573df4d797889aad8a20ed0575da2b44f5d6561/core/scheduler/scheduler.go#L126-L136

So it didn't made sense to call this method too in a go routine.

Reply:
Ok cool, please leave a note there explaining that